modules/nw-ingress-integrating-route-secret-certificate.adoc

```diff
@@ -0,0 +1,79 @@
+//
+// * ingress/routes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-ingress-integrating-route-secret-certificate_{context}"]
+= Securing route with external certificates in TLS secrets
+
+:FeatureName: Securing route with external certificates in TLS secrets
+include::snippets/technology-preview.adoc[]
+
+The following workflow describes how to integrate third-party certificate management solutions, using the certificate reference field outlined under API Extensions.
+
+A new `.spec.tls.externalCertificate` field has been added to the route's `.spec.tls` configuration. This field can be used to specify the name of a secret containing the certificate data.
+
+[NOTE]
+====
+This feature applies to both edge routes and re-encrypt routes.
+====
+
+.Prerequisites
+
+* You have access to the {product-title} cluster as a user with the cluster-admin role.
+
+* You must have generated the serving certificate using third-party systems.
+
+* Ensure the referenced secret contains a valid TLS certificate and key pair in PEM format. The secret must be of type `kubernetes.io/tls` and reside in the same namespace as the route you want to secure. It should include both `tls.key` and `tls.crt` keys within the `data` or `stringData` field of the secret configuration.
+
+* The cluster should have `RouteExternalCertificate` feature gate enabled.
+
+.Procedure
+
+. Create a role in the same namespace as the secret containing the certificate generated, to grant the router service account read access.
++
+[source,terminal]
+----
+$ oc create role secret-reader --verb=get,list,watch --resource=secrets --resource-name=<secret-name> --namespace=<current-namespace> <1>
+----
+<1> Replace <secret-name> with the actual name of your secret.
+
+. Create a rolebinding in the same namespace as the secret and bind the router service account to the newly created role.
++
+[source,terminal]
+----
+$ oc create rolebinding secret-reader-binding --role=secret-reader --serviceaccount=openshift-ingress:router --namespace=<current-namespace> <1>
+----
+<1> Replace <current_namespace> with the namespace where both your secret and route reside.
+
+. Create a route specifying the `.spec.tls.externalCertificate` field to reference the secret containing your certificate:
++
+.Example
+[source,yaml]
+----
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  name: myedge
+  namespace: test
+spec:
+  host: myedge-test.apps.example.com
+  tls:
+    externalCertificate:
+      name: <secret-name>
+    termination: edge
+    ...
+...
```

Comment:
nit: In the preview page, I observed that the last dots are underlined. Is it expected? or some formatting issue?